packages/ibm-products/src/global/js/utils/getNodeTextContent.js

```diff
@@ -42,6 +42,10 @@ export const getNodeTextContent = (node) => {
       return '';
     }
 
+    case 'function': {
+      return getNodeTextContent(node());
```

Comment:
Nice. Please include a test in `getNodeTextContent.test.js` to validate. Thanks.